client/src/Movies/MovieList.js

```diff
@@ -24,7 +25,10 @@ export default class MovieList extends Component {
     return (
       <div className="movie-list">
         {this.state.movies.map(movie => (
+          <Link to={`/movies/${movie.id}`}>
           <MovieDetails key={movie.id} movie={movie} />
```

Comment:
```suggestion
              <MovieDetails key={movie.id} movie={movie} />
```
proper indentation is important too